hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/CapacitySchedulerConfiguration.java

```diff
@@ -531,6 +531,9 @@ public float getMaximumApplicationMasterResourcePerQueuePercent(String queue) {
     		getMaximumApplicationMasterResourcePercent());
   }
 
+  public boolean isMaximumApplicationMasterResourcePerQueuePercentPresent(String queue) {
```

Comment:
isMaximumApplicationMasterResourcePerQueuePercentSet would be a better name for this.